### PR TITLE
ref(quickstart): use markdown-include to de-duplicate quickstart info

### LINKED ISSUE #570: Have one place to install workflow and refer it instead of duplicating

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #570 

azure docs are left alone due to deis/controller#1193.

### CODE REVIEW

File: src/quickstart/provider/azure-acs/install-azure-acs.md [hunk 1]
```diff
@@ -1,85 +1,14 @@
...
 $ export SA_KEY=`az storage account keys list -n $SA_NAME -g RG_NAME --query keys[0].value --output tsv`
-
-```
-
-## Install Deis Workflow
-
-Now that Helm is installed and the repository has been added, install Workflow by running:
-
-```
-$ helm install deis/workflow --namespace=deis --set controller.k8s_api_verify_tls=false,global.storage=azure,azure.accountname=$SA_NAME,azure.accountkey=$SA_KEY,azure.registry_container=registry,azure.database_container=database,azure.builder_container=builder
```
Comment (slack) [inline src/quickstart/provider/azure-acs/install-azure-acs.md]: This command needs to remain the same.

Comment (bacongobbler) [inline src/quickstart/provider/azure-acs/install-azure-acs.md]: Ah yes, the custom Azure setup. Let's leave the Azure docs separate for now until we can get deis/controller#1193 out the door. Nice catch 👍

**Comment by mboersma** [comment]
This LGTM, but @slack should have the last word.

**Comment by slack** [comment]
SGTM!

**Pull request merged**